Panto/css/hero-section.css

```diff
@@ -0,0 +1,91 @@
+.section-hero {
+  background-image: url(../img/hero-img.webp);
+  background-size: cover;
+  background-position: center;
+  height: 108.4rem;
```

Comment:
I've seen that you are using exact values for block-size's and this is generally not a good idea
you should make them more dynamic